### ISSUE TITLE
QuantizeLinear saturates to wrong range for int8

### ISSUE THREAD

**Issue by milkpot** [microsoft/onnxruntime#3442]
**Describe the bug**
QuantizeLinear satures to [-127, 127] for int8.
According to the specification, it satures to [-128, 127].

**Urgency**
None.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04.4 LTS
- ONNX Runtime installed from (source or binary): binary
- ONNX Runtime version: 1.2.0
- Python version: 3.6.9
- Visual Studio version (if applicable): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: 10.1/7.6
- GPU model and memory: GeForce RTX 2080Ti 11GB x2

**To Reproduce**
Run the following code:
```
import numpy as np
import onnx
import onnxruntime
from onnx import helper
from onnx import TensorProto

x = helper.make_tensor_value_info('x', TensorProto.FLOAT, ['N'])
scale = helper.make_tensor_value_info('scale', TensorProto.FLOAT, [])
zero_point = helper.make_tensor_value_info('zero_point', TensorProto.INT8, [])
y = helper.make_tensor_value_info('y', TensorProto.INT8, ['N'])
node = helper.make_node(
    'QuantizeLinear',
    ['x', 'scale', 'zero_point'],
    ['y'],
)
graph = helper.make_graph(
    [node],
    'model',
    [x, scale, zero_point],
    [y],
)
model = helper.make_model(graph)
onnx.save(model, 'model.onnx')

session = onnxruntime.InferenceSession('model.onnx')
result = session.run(['y'], {
    'x': np.array([-129, -128, -127, -126, 126, 127, 128, 129], np.float32),
    'scale': np.array(1, np.float32),
    'zero_point': np.array(0, np.int8),
})
print(result)
```

Result:
```
[array([-127, -127, -127, -126,  126,  127,  127,  127], dtype=int8)]
```

**Expected behavior**
```
[array([-128, -128, -127, -126,  126,  127,  127,  127], dtype=int8)]
```

**Screenshots**
None.

**Additional context**
None.

**Comment by tracysh** [comment]
Thanks for reporting this... the bug has been fixed in #3445.

**Comment by hariharans29** [comment]
Closed via #3445
